docs/architecture.md

```
@@ -14,6 +14,7 @@
 - [Concurrency Model](#concurrency-model)
 - [NSPopover Decisions](#nspopover-decisions)
 - [Design](#design)
+- [E-Tag caching](#e-tag-caching)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)